scripts/install-nix-from-closure.sh

```diff
@@ -40,11 +40,14 @@ elif [ "$(uname -s)" = "Linux" ] && [ -e /run/systemd/system ]; then
 fi
 
 INSTALL_MODE=no-daemon
+DARWIN_APFS_VOLUME=0
 # Trivially handle the --daemon / --no-daemon options
 if [ "x${1:-}" = "x--no-daemon" ]; then
     INSTALL_MODE=no-daemon
 elif [ "x${1:-}" = "x--daemon" ]; then
     INSTALL_MODE=daemon
+elif [ "x${1:-}" = "x--create-volume" ]; then
```

Comment:
This doesn't look like it let's you combine "--daemon" & "--create-volume"? We need an `if` & `shift` here.

Reply:
Also ran into this, given that this will essentially become mandatory the flag isn't required. But I'm inclined to keep it for clarity given all the discussion around encryption.